sympy/stats/tests/test_mix.py

```diff
@@ -18,6 +20,16 @@ def test_density():
     assert simplify(density(N2, Eq(N1, 1))(x)) == \
         sqrt(2)*exp(-(x - 1)**2/8)/(4*sqrt(pi))
 
+def test_MarginalDistribution():
+    a1, p1, p2, p3 = symbols('a1 p1 p2 p3', positive=True)
+    C = Multinomial('C', 3, p1, p2, p3)
+    B = MultivariateBeta('B', a1, C[0])
+    MGR = MarginalDistribution(B, C[0])
+    assert str(MGR.pdf(C)) == \
```

Comment:
what about using `.dummy_eq` or comparing the expressions directly?

Reply:
I tried but it didn't work. :-(

Reply:
Is there any documentation which tells that how SymPy compares expressions? Like, I think knowing the concept will help me in using the right tools. Thanks.

Reply:
There used to be a bug with `.dummy_eq` and summations. Apparently it's still there.